src/zenml/integrations/spark/step_operators/kubernetes_step_operator.py

```diff
@@ -0,0 +1,121 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import os
+from typing import Any, ClassVar, List, Optional, Sequence
+
+from pyspark.conf import SparkConf
+
+from zenml.config.docker_configuration import DockerConfiguration
+from zenml.integrations.spark import SPARK_KUBERNETES_STEP_OPERATOR
+from zenml.integrations.spark.step_operators.spark_step_operator import (
+    SparkStepOperator,
+)
+from zenml.io.fileio import copy
+from zenml.logger import get_logger
+from zenml.repository import Repository
+from zenml.runtime_configuration import RuntimeConfiguration
+from zenml.integrations.spark.step_operators import spark_entrypoint
+from zenml.utils.pipeline_docker_image_builder import PipelineDockerImageBuilder
+from zenml.utils.source_utils import get_source_root_path
+
+logger = get_logger(__name__)
+
+LOCAL_ENTRYPOINT = spark_entrypoint.__file__
+APP_DIR = "/app"
```

Comment:
This is already defined in the pipeline_docker_image_builder.py as the constant `DOCKER_IMAGE_WORKDIR`